mslib/mscolab/models.py

```diff
@@ -157,13 +157,6 @@ def __repr__(self):
                f'last_used: {self.last_used}> '
 
 
-class MessageType(enum.IntEnum):
```

Comment:
that is not so ideal to seperate this from models.py. These classes are used to setup the database.

Reply:
I do not understand?
The only change is that it is in its own file.

Reply:
when one looks on a grouping folder it is clear everything there belongs to models

now we have a models module and one other models module which is not that easy recognized as before, where all was in one file